src/ontology/mondo-edit.obo

```diff
@@ -284615,9 +284615,11 @@ xref: Orphanet:353327 {source="OMIM:610542"}
 xref: Orphanet:590 {source="OMIM:610542"}
 xref: UMLS:C3552335 {source="MONDO:equivalentTo", source="MEDGEN:765249", source="MONDO:MEDGEN"}
 is_a: MONDO:0000182 {source="DC-OMIM:610542", source="OMIM:610542"} ! congenital myasthenic syndrome with tubular aggregates
+is_a: MONDO:0024322 {source="https://orcid.org/0000-0001-6330-7526"} ! disorder of glycosylation
```

Comment:
If you confirmed the SC with OMIM and ICD11, please add these x-ref as a source.

Reply:
added omim and pubmed references